src/Language/PureScript/Publish.hs

```diff
@@ -123,17 +123,21 @@ catchLeft a f = either f pure a
 
 preparePackage' :: PublishOptions -> PrepareM D.UploadedPackage
 preparePackage' opts = do
-  exists <- liftIO (doesFileExist "bower.json")
-  unless exists (userError BowerJSONNotFound)
+  bowerJSONExists <- liftIO (doesFileExist "bower.json")
+  unless bowerJSONExists (userError BowerJSONNotFound)
 
   checkCleanWorkingTree opts
 
   pkgMeta <- liftIO (Bower.decodeFile "bower.json")
                     >>= flip catchLeft (userError . CouldntDecodeBowerJSON)
+  licenseExists <- liftIO (doesFileExist "LICENSE")
+  unless licenseExists (userError LicenseNotFound)
```

Comment:
Maybe we should create a little helper function for checking whether a file exists and throwing a particular error if not?